Dockerfile

```diff
@@ -50,6 +50,7 @@ ADD dist ${base_path}
 
 # change permissions of file inside working dir
 #RUN chown -R ${container_user}:${container_user} ${base_path}/assets/i18n
+RUN chown -R ${container_user}:${container_user} /home/${container_user}
```

Comment:
Not required here 

please add it in line https://github.com/mosip/mosip-compliance-toolkit-ui/blob/ec2b7edaf355fa1d1da6e7cc89b6b7c2e785fdcb/Dockerfile#L40